package.json

```diff
@@ -43,27 +43,27 @@
   "dependencies": {
     "@typescript-eslint/eslint-plugin": "2.0.0",
     "@typescript-eslint/parser": "2.0.0",
-    "babel-eslint": "10.0.2",
-    "eslint-config-prettier": "6.0.0",
+    "babel-eslint": "10.0.3",
+    "eslint-config-prettier": "6.1.0",
     "eslint-plugin-babel": "5.3.0",
-    "eslint-plugin-import": "2.18.0",
-    "eslint-plugin-jest": "22.11.1",
+    "eslint-plugin-import": "2.18.2",
+    "eslint-plugin-jest": "22.15.2",
     "eslint-plugin-jsx-a11y": "6.2.3",
     "eslint-plugin-node": "9.1.0",
     "eslint-plugin-prettier": "3.1.0",
     "eslint-plugin-promise": "4.2.1",
-    "eslint-plugin-react": "7.14.2",
-    "eslint-plugin-react-hooks": "1.6.1",
-    "eslint-plugin-unicorn": "9.1.1"
+    "eslint-plugin-react": "7.14.3",
+    "eslint-plugin-react-hooks": "2.0.1",
```

Comment:
This was technically a breaking change. https://github.com/facebook/react/pull/16455/files

Reply:
we might not, internally, but this is a public package